login.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+	session_start();
+	
+	header('Content-type: application/json');
+	
+	$inData = getRequestInfo();
+	
+	$id = 0;
+	$firstName = "";
+	$lastName = "";
+	
+	$database_url = parse_url(getenv('DATABASE_URL'));
+
+	$server = $database_url['host'];
+	$user = $database_url['user'];
+	$pass = $database_url['pass'];
+	$db = substr($database_url['path'], 1);
+	
+	$conn = new mysqli($server, $user, $pass, $db);
+	if($conn->connect_error)
+	{
+		returnWithError($conn->connect_error);
+	}
+	else
+	{
+		$stmt = $conn->prepare("SELECT ID,firstName,lastName FROM Users WHERE Login=? AND Password =?");
+		$stmt->bind_param("ss", $inData["login"], $inData["password"]);
+		$stmt->execute();
+		$result = $stmt->get_result();
+		
+		if ($row = $result->fetch_assoc())
+		{
+			$_SESSION["id"] = $row['ID'];
+			$_SESSION["firstName"] = $row['firstName'];
```

Comment:
This is good but I think we should only really store the id in our session variable. Also make sure to update the `$row['ID']` to match the ERD (remember case sensitive).